pom.xml

```diff
@@ -152,67 +130,6 @@
           </archive>
         </configuration>
       </plugin>
-      <plugin>
-        <artifactId>maven-shade-plugin</artifactId>
```

Comment:
As far as I understand, this plugin is used to relocate the eclipse runtime into a different package and to incorporate the moved classes in an uber jar, instead of keeping it as an external dependency. Sounds to me like we will get a new additional dependency in cplace then.

Reply:
> Sounds to me like we will get a new additional dependency in cplace then.
What do you mean by that? I removed this because I thought that we don't need the uber jar. I think that they use it for running the tool standalone but we are embedding it into cplace

Reply:
Looking into the existing `daisydiff-no-xerces-1.2.jar` file, I see packages `org.eclipse.core.runtime` and `org.cyberneko.html` next to `org.outerj.daisy.diff`.

I don't know this plugin, merely scanned its documentation page. I understood that this plugin is used to bundle the `eclipse-core-runtime` dependency into the same jar file with a relocated package for some dependencies. Further down in this POM, I see the `maven-assembly-plugin`, which could as well be the one producing our artifact. 🤔 

Basically, I'd like you to check the resulting artifact from this branch, to see what it contains and what its dependencies are, compared to the library we currently use. :-)

Reply:
```
~/projects/cplace-daisydiff/target (feature/remove_xerces_dependency*) jar tf daisydiff-1.2.0-SNAPSHOT.jar                                                                                       130 ↵ joachimlengacher@Joachims-MBP
META-INF/
META-INF/MANIFEST.MF
l10n/
org/
org/eclipse/
org/eclipse/compare/
org/eclipse/compare/internal/
org/eclipse/compare/rangedifferencer/
org/outerj/
org/outerj/daisy/
org/outerj/daisy/diff/
org/outerj/daisy/diff/output/
org/outerj/daisy/diff/html/
org/outerj/daisy/diff/html/modification/
org/outerj/daisy/diff/html/ancestor/
org/outerj/daisy/diff/html/ancestor/tagtostring/
org/outerj/daisy/diff/html/dom/
org/outerj/daisy/diff/html/dom/helper/
org/outerj/daisy/diff/helper/
org/outerj/daisy/diff/tag/
xslfilter/
l10n/messages_fr.properties
l10n/messages_en.properties
l10n/messages.properties
org/eclipse/compare/internal/LCSSettings.class
org/eclipse/compare/internal/LCS.class
org/eclipse/compare/internal/CompareMessages.class
org/eclipse/compare/rangedifferencer/OldDifferencer.class
org/eclipse/compare/rangedifferencer/RangeDifferencer.class
org/eclipse/compare/rangedifferencer/OldDifferencer$LinkedRangeDifference.class
org/eclipse/compare/rangedifferencer/RangeComparatorLCS.class
org/eclipse/compare/rangedifferencer/IRangeComparator.class
org/eclipse/compare/rangedifferencer/RangeDifference.class
org/eclipse/compare/rangedifferencer/DifferencesIterator.class
org/outerj/daisy/diff/XslFilter.class
org/outerj/daisy/diff/output/TextDiffer.class
org/outerj/daisy/diff/output/Differ.class
org/outerj/daisy/diff/output/TextDiffOutput.class
org/outerj/daisy/diff/output/DiffOutput.class
org/outerj/daisy/diff/html/HtmlSaxDiffOutput.class
org/outerj/daisy/diff/html/HTMLDiffer.class
org/outerj/daisy/diff/html/modification/ModificationType$4.class
org/outerj/daisy/diff/html/modification/HtmlLayoutChange$Type.class
org/outerj/daisy/diff/html/modification/ModificationType$2.class
org/outerj/daisy/diff/html/modification/HtmlLayoutChange.class
org/outerj/daisy/diff/html/modification/ModificationType$5.class
org/outerj/daisy/diff/html/modification/ModificationType$1.class
org/outerj/daisy/diff/html/modification/ModificationType$3.class
org/outerj/daisy/diff/html/modification/ModificationType.class
org/outerj/daisy/diff/html/modification/Modification.class
org/outerj/daisy/diff/html/TextNodeComparator.class
org/outerj/daisy/diff/html/ancestor/TextOnlyComparator.class
org/outerj/daisy/diff/html/ancestor/TagChangeSematic$1.class
org/outerj/daisy/diff/html/ancestor/TagChangeSematic.class
org/outerj/daisy/diff/html/ancestor/AncestorComparator.class
org/outerj/daisy/diff/html/ancestor/ChangeTextGenerator.class
org/outerj/daisy/diff/html/ancestor/TagChangeSematic$2.class
org/outerj/daisy/diff/html/ancestor/AncestorComparatorResult.class
org/outerj/daisy/diff/html/ancestor/ChangeText.class
org/outerj/daisy/diff/html/ancestor/tagtostring/AnchorToString.class
org/outerj/daisy/diff/html/ancestor/tagtostring/NoContentTagToString.class
org/outerj/daisy/diff/html/ancestor/tagtostring/TagToString.class
org/outerj/daisy/diff/html/ancestor/tagtostring/TagToStringFactory.class
org/outerj/daisy/diff/html/dom/WhiteSpaceNode.class
org/outerj/daisy/diff/html/dom/ImageNode.class
org/outerj/daisy/diff/html/dom/TextNode.class
org/outerj/daisy/diff/html/dom/TagNode.class
org/outerj/daisy/diff/html/dom/DomTreeBuilder.class
org/outerj/daisy/diff/html/dom/SeparatingNode.class
org/outerj/daisy/diff/html/dom/BodyNode.class
org/outerj/daisy/diff/html/dom/helper/AttributesMap.class
org/outerj/daisy/diff/html/dom/helper/LastCommonParentResult.class
org/outerj/daisy/diff/html/dom/Node.class
org/outerj/daisy/diff/html/dom/DomTree.class
org/outerj/daisy/diff/HtmlCleaner.class
org/outerj/daisy/diff/DaisyDiff.class
org/outerj/daisy/diff/Main.class
org/outerj/daisy/diff/helper/SaxBuffer$StartEntity.class
org/outerj/daisy/diff/helper/SaxBuffer$StartPrefixMapping.class
org/outerj/daisy/diff/helper/SaxBuffer$StartDocument.class
org/outerj/daisy/diff/helper/SaxBuffer$Characters.class
org/outerj/daisy/diff/helper/SaxBuffer$EndDTD.class
org/outerj/daisy/diff/helper/NekoHtmlParser.class
org/outerj/daisy/diff/helper/NekoHtmlParser$RemoveNamespacesHandler.class
org/outerj/daisy/diff/helper/SaxBuffer$SkippedEntity.class
org/outerj/daisy/diff/helper/SaxBuffer$StartDTD.class
org/outerj/daisy/diff/helper/SaxBuffer$IgnorableWhitespace.class
org/outerj/daisy/diff/helper/SaxBuffer$EndDocument.class
org/outerj/daisy/diff/helper/SaxBuffer$SaxBit.class
org/outerj/daisy/diff/helper/MergeCharacterEventsHandler.class
org/outerj/daisy/diff/helper/SaxBuffer.class
org/outerj/daisy/diff/helper/SaxBuffer$StartElement.class
org/outerj/daisy/diff/helper/SaxBuffer$EndElement.class
org/outerj/daisy/diff/helper/SaxBuffer$EndPrefixMapping.class
org/outerj/daisy/diff/helper/SaxBuffer$StartCDATA.class
org/outerj/daisy/diff/helper/SaxBuffer$Comment.class
org/outerj/daisy/diff/helper/SaxBuffer$EndCDATA.class
org/outerj/daisy/diff/helper/SaxBuffer$PI.class
org/outerj/daisy/diff/helper/SaxBuffer$EndEntity.class
org/outerj/daisy/diff/tag/TextAtom.class
org/outerj/daisy/diff/tag/TagAtom.class
org/outerj/daisy/diff/tag/TagSaxDiffOutput.class
org/outerj/daisy/diff/tag/TagComparator.class
org/outerj/daisy/diff/tag/Atom.class
org/outerj/daisy/diff/tag/IAtomSplitter.class
org/outerj/daisy/diff/tag/DelimiterAtom.class
org/outerj/daisy/diff/tag/ArgumentComparator.class
org/outerj/daisy/diff/tag/TagDiffer.class
xslfilter/htmlheader.xsl
xslfilter/tagheader.xsl
xslfilter/cleanup.xsl
META-INF/maven/
META-INF/maven/cf.cplace.org.outerj.daisy/
META-INF/maven/cf.cplace.org.outerj.daisy/daisydiff/
META-INF/maven/cf.cplace.org.outerj.daisy/daisydiff/pom.xml
META-INF/maven/cf.cplace.org.outerj.daisy/daisydiff/pom.properties

```

Reply:
It's only the stuff that is part of the project. Not an uber JAR.